data/org.freedesktop.portal.Settings.xml

```diff
@@ -43,6 +43,13 @@
           Unknown values should be treated as 0 (no preference).
         </para></listitem>
       </varlistentry>
+      <varlistentry>
+        <term>org.freedesktop.appearance accent-color (uuu)</term>
```

Comment:
Any particular reason why limiting color values to 8 bit per channel? Changing to `(ddd)` and changing the range to 0.0 - 1.0 makes it future proof and able to describe all values in e.g. fp16 pixel formats.

Reply:
That's what we had originally. I don't see any reason to not change it to doubles, so I'll do that now.